Sources/FoundationEssentials/Data/Data.swift

```diff
@@ -604,6 +605,7 @@ internal final class __DataStorage : @unchecked Sendable {
 
 @frozen
 @available(macOS 10.10, iOS 8.0, watchOS 2.0, tvOS 9.0, *)
+@_addressableForDependencies
```

Comment:
Any ABI or source stability impact for this?

Reply:
There's no source stability impact. It only changes the mangling for functions that try to establish a lifetime dependence on `Data`, and there are none of those until we publish these `span` properties.